src/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/ECDsaOpenSsl.ImportExport.cs

```diff
@@ -0,0 +1,270 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Security.Cryptography
+{
+    public sealed partial class ECDsaOpenSsl : ECDsa
+    {
+        /// <summary>
+        ///         ImportParameters will replace the existing key that ECDsaOpenSsl is working with by creating a
+        ///         new CngKey. If the parameters structure contains only Q,
+        ///         then only a public key will be imported. If the parameters also contains D, 
+        ///         then a full key pair will be imported. If the parameters do not specify Q,
+        ///         a then a full key pair will be created.
```

Comment:
Yeah this is a discussion topic. If you have an explicit curve an want to generate a new key, how would you do that? Relates to the GenerateKey(curve) discussion (take offline)